.github/WORKFLOWS.md

```diff
@@ -0,0 +1,40 @@
+# GitHub-workflows in Nessie
```

Comment:
Could this be `README.md` so its visible on the site? OR is this a special keyword for the .github directory

Reply:
Done